### PR TITLE
MBS-8292 CompositeBuild

### PR DESCRIPTION
**Pull request by sergioboi**

### CODE REVIEW

**Review by dsvoronin — APPROVED** [review]

File: build.sh [hunk 1]
```diff
@@ -8,9 +8,9 @@ source $(dirname $0)/_main.sh
...
 docs/check.sh
 
-TEMP_PROJECT_VERSION="temp-version"
 # `tasks` triggers full tasks graph resolving, checking for possible misconfigurations
 runInBuilder "set -e;
     ./gradlew help;
-    ./gradlew build publishToMavenLocal ${GRADLE_ARGS} -PprojectVersion=${TEMP_PROJECT_VERSION};
-    ./gradlew tasks :subprojects:android-test:test-app:instrumentationUi ${GRADLE_ARGS} -PinfraVersion=${TEMP_PROJECT_VERSION}"
+    ./gradlew help -PuseCompositeBuild=false;
+    ./gradlew -p subprojects build ${GRADLE_ARGS};
```
Comment (eugene-krivobokov) [inline build.sh:15]: What is `-p`?

Comment (sergioboi) [reply build.sh:15]: project path

File: gradle.properties [hunk 2]
```diff
@@ -4,10 +4,6 @@ org.gradle.parallel=true
 org.gradle.caching=true
 # mandatory from AGP 3.6
 android.useAndroidX=true
-# The next version of all artifacts
-# Scheme: year.month[.<version>]
-# Version is just a consecutive number. We don't need to bound releases to days.
-projectVersion=2020.5.6
 # Current stable version to be used in this project
 infraVersion=2020.5.5
```
Comment (eugene-krivobokov) [inline gradle.properties:8]: Do we need to fix [publishing instructions](https://avito-tech.github.io/avito-android/docs/infrastructure/#publishing-a-new-release)?
I didn't get when to change and publish a new version

Comment (sergioboi) [reply gradle.properties:8]: Done

**Pull request merged**